src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.test.tsx

```diff
@@ -0,0 +1,48 @@
+import { type RenderResult, render, screen } from '@testing-library/react'
+import ArrivalByTimeChart from './ArrivalByTimeChart'
+import '@testing-library/jest-dom'
+import type { ReactElement } from 'react'
+import testBusData from './testdata/data.json'
+
+describe('ArrivalByTimeChart', () => {
+  let renderedComponent: RenderResult
+
+  vi.mock('recharts', async () => {
+    const mod: typeof import('recharts') = await vi.importActual('recharts')
+    return {
+      ...mod,
+      ResponsiveContainer: ({ children }: { children: ReactElement }) => (
+        <mod.ResponsiveContainer height={300} aspect={1}>
+          {children}
+        </mod.ResponsiveContainer>
+      ),
+    }
+  })
+
+  beforeEach(() => {
+    renderedComponent = render(
+      <ArrivalByTimeChart data={testBusData} operatorId={testBusData[0].id} />,
+    )
+  })
+
+  test('renders without crashing', () => {
+    expect(screen.getByText(testBusData[0].name)).toBeInTheDocument()
+  })
+
+  test('tooltip wrapper exists', () => {
+    expect(renderedComponent.container.querySelector('.recharts-tooltip-wrapper')).not.toBeNull()
```

Comment:
thanks! that's much better.
I think last time you forgot to add `()`, and the `toBeNull` didn't assert anything. It's so nice of you to get it work like that.

Reply:
Thanks! 
it was all thanks to you fixing it in the previous PR, wouldn't have seen it otherwise!

Reply:
you would have see it if the lint was fixed earlier :sweat_smile: